tests/api-tests/fields/types/fixtures/text/test-fixtures.ts

```diff
@@ -4,6 +4,7 @@ export const name = 'Text'
 export const typeFunction = text
 export const exampleValue = () => 'foo'
 export const exampleValue2 = () => 'bar'
+export const nonNullableDefault = true
```

Comment:
By adding this for `multiselect`, and `Text` supporting `null` in https://github.com/keystonejs/keystone/pull/9057, `text` is now _less_ special in our tests